sound/soc/sof/intel/hda-stream.c

```diff
@@ -36,57 +36,97 @@
 #include "hda.h"
 
 /*
- * set up Buffer Descriptor List (BDL) for host memory transfer
- * BDL describes the location of the individual buffers and is little endian.
+ * set up one of BDL entries for a stream
```

Comment:
Why the change, there is no explanation in the commit message. What is wrong with existing BDL func ?

Reply:
in the existing BDL, we don't support interrupt mode with DIPB/posbuf, let me add commit message to explain it.